cmd/okctl/forward_postgres.go

```diff
@@ -102,6 +105,19 @@ func buildForwardPostgres(o *okctl.Okctl) *cobra.Command {
 			},
 		),
 		RunE: func(_ *cobra.Command, _ []string) (err error) {
+			services, err := o.ClientServices(o.StateHandlers(o.StateNodes()))
```

Comment:
Totally off topic: This code we need to use to get services is just so .. improvable :smile:

Reply:
xD /giphy nodding furiously